### ISSUE TITLE
Opens iTerm but does not navigate to folder, and does not focus on iTerm

### ISSUE THREAD

**Issue by garyking** [wbond/sublime_terminal#132]
The updated package opens iTerm 2 but does not navigate to the project's folder, and does not focus on iTerm. "activate" calls are probably needed, like what I have below.

Settings:

```
{
  "terminal": "iTerm2-v3.sh",
  "parameters": ["--open-in-tab"]
}
```

[This works for me](https://gist.github.com/garyking/d4ebfd811ca171624aa99c81f398af2e).

**Comment by twolfson** [comment]
Can you submit a PR with the appropriate fixes?

**Comment by garyking** [comment]
https://github.com/wbond/sublime_terminal/pull/133

**Comment by twolfson** [comment]
This has been resolved by #133 and released in `1.14.0`
